py3-pip.yaml

```diff
@@ -0,0 +1,82 @@
+package:
+  name: py3-pip
+  version: "24.0"
+  epoch: 2
+  description: The PyPA recommended tool for installing Python packages.
+  copyright:
+    - license: MIT
+  dependencies:
+    provider-priority: 0
+
+vars:
+  pypi-package: pip
+
+data:
+  - name: py-versions
+    items:
+      3.10: "310"
+      3.11: "311"
+      3.12: "312"
```

Comment:
@smoser pondering if we should change the range to just "10 11 12". and prefix with python3. or 3 as needed..... but maybe that's still better as easier to extend to if/when python4 happens (i hope never).

Reply:
i think leave it sa 3.XX and 3XX. unless you feel strongly otherwise.
i dont' think python4 is a near term concern.